kayvee.go

```diff
@@ -4,6 +4,14 @@ import (
 	"encoding/json"
 )
 
+// Log Levels:
+const Unknown = "unknown"
```

Comment:
Thoughts on giving this its own type, as well? Something like:

```go
type LogLevel string

const Unknown LogLevel = "unknown"
const Critical LogLevel = "critical"
const Error LogLevel = "error"
const Warning LogLevel = "warning"
const Info LogLevel = "info"
const Trace LogLevel = "trace"
```

Downside: changing the functions to take in this type instead of a string would be a breaking change